modules/api/src/main/UserApi.scala

```diff
@@ -112,7 +114,9 @@ private[api] final class UserApi(
                   "followable" -> followable,
                   "following" -> relation.has(true),
                   "blocking" -> relation.has(false),
-                  "followsYou" -> isFollowed
+                  "followsYou" -> isFollowed,
+                  "twitchStream" -> streamer.flatMap(_.twitch.map(_.minUrl)),
+                  "youTubeStream" -> streamer.flatMap(_.youTube.map(_.minUrl))
```

Comment:
This would be more extensible:
```
"stream" -> streamer.map { s =>
  Json.obj(
    "twitch" -> s.twitch.map { t =>
      Json.obj("url" -> t.minUrl)
    },
    ...
  )
}
```